x-pack/plugin/ent-search/qa/rest/roles.yml

```diff
@@ -22,15 +22,11 @@ admin:
         "test-re-creating-search-application",
         "test-search-application-to-delete",
         "test-nonexistent-search-application",
-        "test-search-application-with-invalid-template"
+        "test-search-application-with-invalid-template",
+        "test-search-index1",
```

Comment:
Nitpick: Should we group the indices and applications together (e.g. move these two indices to line 15?)

Reply:
sure https://github.com/elastic/elasticsearch/pull/96176/commits/b97893eed96633cbd040f8009b0d1ee2bba89fb2